api/src/main/java/com/microsoft/gctoolkit/jvm/LoggingDiary.java

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
Are you going to resolve the overlap between this an JVMConfiguration?

Reply:
I'll likely bind this method to the appropriate JVMConfiguration class